db/system_distributed_keyspace.hh

```diff
@@ -44,6 +45,7 @@ public:
 
     static constexpr auto VIEW_BUILD_STATUS = "view_build_status";
     static constexpr auto SERVICE_LEVELS = "service_levels";
+    static constexpr auto SERVICE_LEVELS_V2 = "service_levels_v2";
```

Comment:
I don't have any particular problem with this name, but just for completeness sake, why did you choose to go with a different table name?